dashboard-ui/src/components/Home/scoreChart.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { ResponsiveBar } from '@nivo/bar';
+import gql from 'graphql-tag';
+import { Query } from 'react-apollo';
+
+const GET_HISTORIES_BY_ID = "getAllHistoryByID";
+const get_all_history_by_id = gql`
+    query getAllHistoryByID($historyId: ID) {
+        getAllHistoryByID(historyId: $historyId)
+    }`;
+
+const formatLeftIndexString = function(peformerADMString) {
+    if(peformerADMString.indexOf("ALIGN-ADM") > -1 ) {
+        return ("Kitware: " + peformerADMString);
+    } else if (peformerADMString.indexOf("TAD") > -1){
+        return ("Parallax: " + peformerADMString);
+    } else {
+        return peformerADMString;
+    }
+}
+
+const MyResponsiveBar = ({mappedArray}) => {
+    return (
+        <div className="score-chart-container">
+            <ResponsiveBar
+                data={mappedArray}
+                keys={['Average Alignment Score']}
+                indexBy="performer"
+                margin={{ top: 50, right: 50, bottom: 50, left: 160 }}
+                padding={0.4}
+                layout="horizontal"
+                valueScale={{ type: 'linear', min: 0, max: 1 }}
+                indexScale={{ type: 'band', round: true }}
+                colors={['#00A0D2', "#999D5D"]}
+                colorBy="index"
+                borderColor={{ from: 'color', modifiers: [['darker', 1.6]] }}
+                borderWidth={0.2}
+                axisTop={null}
+                axisRight={null}
+                axisLeft={{ format: v => formatLeftIndexString(v) }}
+                axisBottom={{
+                    tickSize: 5,
+                    tickPadding: 5,
+                    tickRotation: 0,
+                    legend: 'Alignment Score',
+                    legendPosition: 'middle',
+                    legendOffset: -10,
+                }}
+                labelSkipWidth={12}
+                labelSkipHeight={12}
+                labelTextColor="black"
+                animate={true}
+                motionStiffness={90}
+                motionDamping={15}
+            />
+        </div>
+    );
+};
+
+class ScoreChart extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = { 
+            style: {fontFamily: "Lato", textAlign: "center"}
```

Comment:
So it would be interesting to see if this style is being used, because, I see we are using state.styles so perhaps both can be removed, or perhaps both used, either way, we should check that out.